servers/wanaku-server-quarkus/src/main/java/ai/wanaku/server/quarkus/api/v1/resources/ResourcesResource.java

```diff
@@ -45,18 +47,13 @@ public Response expose(ResourceReference resource) {
     @Path("/list")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @APIResponses({
-            @APIResponse(responseCode = "200", content = @Content(
-                    schema = @Schema(type = SchemaType.ARRAY, implementation = ResourceReference.class))),
-            @APIResponse(responseCode = "500", content = @Content(schema = @Schema(implementation = String.class)))
-    })
-    public Response list() {
+    public RestResponse<WanakuResponse<List<ResourceReference>>> list() {
         try {
             List<ResourceReference> list = resourcesBean.list();
-            return Response.ok(list).build();
+            return RestResponse.ok(new WanakuResponse<>(list));
```

Comment:
@Croway will you do the same for Tools list api?

Reply:
sure, I'll think a bit more about the available options, and then I'll update all the pages.